lib/turnsole/heliotrope/service.rb

```diff
@@ -290,6 +306,78 @@ def unsubscribe_product_institution(product_identifier:, institution_identifier:
         response.success?
       end
 
+      #
+      # License { :full, :trial, :none }
+      #
+      #   :full - both read and download access
+      #   :trial - read access (no download access)
```

Comment:
Change :trial to :read to match the nomenclature being used.

Reply:
Yes, missed it, fixed now.